packages/tachyons/src/core/_outlines.css

```diff
@@ -1,6 +1,7 @@
-.outline {
+/* Clash with outline in orbit ui */
```

Comment:
What do we do with this?

If I remember correctly this is a class that we provide for the apps.

Reply:
Yes but it clashes with .ui....outline, I am pretty sure it is not used in Apps. I'd rather have them set it on their side if needed. Thoughts?

Reply:
Would be safest way to handle this until we know more about it, so sure.

Reply:
When in a Tag but I guess 16px wouldn't be bad either

Reply:
I'll remove it and note in release notes.